app/assets/stylesheets/components/top-bar.scss

```diff
@@ -4,50 +4,86 @@ $main-tiles: 7;
 .top-bar {
   @include outer-container;
   max-width: 100%;
-  background: image-url('background-pattern.jpg') 0 3px;
   clear: both;
+  position:relative;
+  top:-2px;
 
   div {
-    display: inline;
     margin: 0;
     padding: 0;
   }
 }
 
-.skyline {
-  margin-top: 1px;
-}
 
 .top-bar__logo {
-  float: left;
+border-top:1px solid #e9e3c8;
+text-align:center;
+background:url("skyline2.png") repeat-x;
+background-size:1100px;
+background-position:50% 60px;
+    img{
+    	margin:20px auto 20px auto;
+    	width:700px;
+    }
+    
+    @media screen and (max-width: 700px) {
+    	background-image:none;
+    
+    }
+  
 }
 
 .top-bar__links {
-  float: right;
-
-  img {
-    @media screen and (max-width: 920px) {
-      display: none;
-    }
-  }
-
-  nav {
-    bottom: 5px;
+text-align:center;
```

Comment:
Can you make sure your indentation here lines up with the indentation in the rest of the file? It should be more like:

```css
.top-bar__links {
  text-align: center
}
```